starknet_py/net/client_models.py

```diff
@@ -32,6 +32,16 @@ class Event:
     data: List[int]
 
 
+@dataclass
+class Events:
```

Comment:
I do not like the name :hmmm:
should be something like EventsResponse, because continuation_token is here
and `events.events` looks weird

cc @cptartur